crates/db_views/src/post_view.rs

```diff
@@ -219,15 +219,15 @@ impl<'a> PostQuery<'a> {
 
     let mut query = post::table
       .inner_join(person::table)
-      .inner_join(community::table)
+      .inner_join(post_aggregates::table)
+      .inner_join(community::table.on(post_aggregates::community_id.eq(community::id)))
       .left_join(
         community_person_ban::table.on(
```

Comment:
That means this one here, and others should also join to `post_aggregates.community_id`